README.md

```diff
@@ -1,7 +1,7 @@
 lshw: HardWare LiSter for Linux
 ===============================
 
-lshw is a small tool to provide detailed information on the hardware configuration of the machine. It can report exact memory configuration, firmware version, mainboard configuration, CPU version and speed, cache configuration, bus speed, etc. on DMI-capable x86 or EFI (IA-64) systems and on some ARM and PowerPC machines (PowerMac G4 is known to work).
+lshw is a small tool to provide detailed information on the hardware configuration of the machine. It can report exact memory configuration, firmware version, mainboard configuration, CPU version and speed, cache configuration, bus speed, etc. on DMI-capable x86 or EFI (IA-64) systems and LoongArch and on some ARM and PowerPC machines (PowerMac G4 is known to work).
```

Comment:
```suggestion
lshw is a small tool to provide detailed information on the hardware configuration of the machine. It can report exact memory configuration, firmware version, mainboard configuration, CPU version and speed, cache configuration, bus speed, etc. on DMI-capable x86 or EFI (IA-64, LoongArch and others) systems and on some ARM and PowerPC machines (PowerMac G4 is known to work).
```

Reply:
已经更正，谢谢